src/components/base/Input/index.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component, Fragment} from 'react';
+
+import cx from 'classnames';
+
+class Input extends Component {
+  inputRef = React.createRef();
+
+  state = {
+    showCopySuccessIndicator: false
+  }
+
+  handleCopy = () => {
+    const input = this.inputRef.current;
```

Comment:
kind of a nitpick but you don't actually need this `const`, since you're only using it one time